apps/web/components/apps/AppPage.tsx

```diff
@@ -230,6 +229,7 @@ export const AppPage = ({
                 handleAppInstall();
               },
               loading: isLoading,
+              disabled: disableInstall || props.disabled,
```

Comment:
[It seems like _disableInstall_ is translated to _props.disabled_](https://github.com/calcom/cal.com/blob/12-03-feat_add_event_handling_for_availability_and_booking_events/packages/app-store/InstallAppButtonWithoutPlanCheck.tsx#L25) and thus we don't need to use disableInstall variable from outside the component. 

We need to figure out what case is there where disableInstall isn't mapped to props.disabled